runelite-client/src/main/java/net/runelite/client/plugins/microbot/fishing/minnows/MinnowsScript.java

```diff
@@ -0,0 +1,69 @@
+package net.runelite.client.plugins.microbot.fishing.minnows;
+
+import net.runelite.api.NPC;
+import net.runelite.api.coords.WorldArea;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.client.plugins.microbot.Microbot;
+import net.runelite.client.plugins.microbot.Script;
+import net.runelite.client.plugins.microbot.util.npc.Rs2Npc;
+import net.runelite.client.plugins.microbot.util.player.Rs2Player;
+
+import java.util.concurrent.TimeUnit;
+
+public class MinnowsScript extends Script {
+    public static final WorldArea MINNOWS_PLATFORM = new WorldArea(new WorldPoint(2607, 3440, 0), 2622 - 2607, 3446 - 3440);
+    private static final int FLYING_FISH_GRAPHIC_ID = 1387;
+    private static final int FISHING_SPOT_1_ID = 7732;
+    private static final int FISHING_SPOT_2_ID = 7733;
+    public static double version = 1.0;
```

Comment:
Make version a string, and give it a MAJOR.MINOR.PATCH.


Major Version (X.0.0): Introduces significant changes or new features that may break backward compatibility with previous versions.
Minor Version (0.X.0): Adds new, backward-compatible functionality and improvements without breaking existing code.
Patch Version (0.0.X): Provides backward-compatible bug fixes and minor improvements, ensuring stability and performance.